annsa/tests/test_readspectra.py

```diff
@@ -0,0 +1,14 @@
+import os
+from annsa.annsa import read_spectrum
+
+
+def test_read_spectrum1():
+    TESTDATA_FILENAME = os.path.join(os.path.dirname(__file__), './data_folder/rocky_flats_spectra.spe')
+    _ = read_spectrum(TESTDATA_FILENAME)
+    pass
+
+
+def test_read_spectrum2():
```

Comment:
```suggestion
def test_read_spectrum_gadras_template():
```

Reply:
Same comment as above. 
I recommend the more meaningful name, above.
More importantly, though, this test simply tests whether this function runs without throwing an exception.
This really isn't much of a test. Since the exception would be thrown at runtime, it's actually the kind of test that's auto-caught by using the package.
A real test would check that the resulting object, `_`, has the features you expect, given you know the file it's read.

(Related... why are variables that are plain underscores used everywhere in this codebase? It's unusual to me.)